src/pages/Trends/utils.js

```diff
@@ -0,0 +1,36 @@
+const defaultMoveFn = (target, value) => target < value
+const defaultCheckFn = (target, value) => target === value
+
+export const binarySearch = ({
```

Comment:
@DmitriVanGuard I think I saw before bs function in our codebase

Reply:
@Partysun I have [found it](https://github.com/santiment/app/blob/c017a705077988430e006875cbfad5d35f60b952/src/utils/utils.js#L87). It's currently optimized for finding correct history price (basically for datetimes) and I'm a little bit afraid to modify it now using new `binarySearch` function (which is general and can be used in whatever case). I will do it in a separate PR after merging this one